geoips/commandline/test_interfaces.py

```diff
@@ -76,52 +78,30 @@ def test_deprecated_interfaces(
 
 def main():
     """Script to test all dev and stable interfaces."""
-
     failed_interfaces = []
     failed_plugins = []
     successful_interfaces = []
     successful_plugins = []
 
-    # Test all the "old" interfaces using the original logic.
```

Comment:
Oops, this should *not* have been commented out.  I have added it back in so we can continue to test the "old" interfaces until they are all switched over.